Library/ViewModels/RewardPledgeViewModel.swift

```diff
@@ -559,9 +574,19 @@ RewardPledgeViewModelOutputs {
       changePaymentMethodEvent.errors()
     )
 
-    self.showAlert = pledgeErrors
-      .map { $0.errorEnvelope.errorMessages.first }
-      .skipNil()
+    self.showAlert = Signal.merge(
+      pledgeErrors
+        .map { error in
+          let shouldDismiss = (error.errorEnvelope.errorMessages.first == nil
+            || error.errorEnvelope.ksrCode == .UnknownCode)
+          return (message: error.errorEnvelope.errorMessages.first ?? Strings.general_error_something_wrong(),
+                  shouldDismiss: shouldDismiss)
```

Comment:
im not sure i understand why we'd want `shouldDismiss: true` for pledge errors. those are typically fixable errors other than not having internet